store/views.py

```diff
@@ -77,6 +93,54 @@ def loanBookView(request):
 @csrf_exempt
 @login_required
 def returnBookView(request):
-    pass
+    book_id = request.POST.get("bid")
+    book = BookCopy.objects.get(pk=book_id)
+    if(book):
+      message ="success"
+      book.status = True
+      book.borrower = None
+      book.borrow_date = None
+      book.save()
+    else :
+      message = "failure" 
+    response_data = {
+        'message': message,
+    }
+    return JsonResponse(response_data)
 
 
+@csrf_exempt
+@login_required
+def userBookRating(request):
+    response_data = {
+        'message': "failure",
+        'rating' : 0 
+    }
+    prating = '0'
+    userid=request.user.id
+    data = request.POST
+    rating =data.get('rating')
+    bookid = data.get('bid')
+    print(rating)
+    if BookRating.objects.filter(user_id=userid).filter(book_id=bookid).count() == 1:
+        prating = BookRating.objects.filter(user_id=userid).filter(book_id=bookid).first().rating
+        print('rating')
```

Comment:
You've not put a backend validation on the rating, so the user can simply edit the JS code you've written in the template and easily put invalid values of rating.